bot/cogs/queue.py

```diff
@@ -32,41 +14,36 @@ def __init__(self, bot):
         """ Set attributes. """
         self.bot = bot
         self.guild_queues = {}  # Maps Guild: QQueue
+        self.last_queue_msgs = {}
 
-    @commands.Cog.listener()
-    async def on_ready(self):
-        """ Initialize an empty list for each guild the bot is in. """
-        for guild in self.bot.guilds:
-            if guild not in self.guild_queues:  # Don't add empty queue if guild already loaded
-                self.guild_queues[guild] = QQueue()
-
-    @commands.Cog.listener()
-    async def on_guild_join(self, guild):
-        """ Initialize an empty list for guilds that are added. """
-        self.guild_queues[guild] = QQueue()
-
-    @commands.Cog.listener()
-    async def on_guild_remove(self, guild):
-        """ Remove queue list when a guild is removed. """
-        self.guild_queues.pop(guild)
-
-    def queue_embed(self, guild, title=None):
+    async def queue_embed(self, guild, title=None):
         """ Method to create the queue embed for a guild. """
```

Comment:
Can you explain what a `queue embed` is exactly? The naming of this is a little confusing

Reply:
Ahh you mean an embed as in a discord message. Nevermind